docs/tasks/istio-installation.md

```diff
@@ -1,37 +1,105 @@
 ---
-title: Installing Istio
-headline: Installing Istio
+title: Istio Installation
+headline: Istio Installation
 sidenav: doc-side-tasks-nav.html
 bodyclass: docs
 layout: docs
 type: markdown
 ---
 {% capture overview %}
-This task shows how to install Istio in a Kubernetes cluster. You'll learn
+This page shows how to get started using Istio in a Kubernetes cluster. You'll learn
 how to install Istio and get it initially configured and running.
 {% endcapture %}
 
 {% capture prerequisites %}
-* Do this.
-* Do this too.
+* The following instructions assume you have access to a Kubernetes cluster. To install Kubernetes locally, try [minikube](https://github.com/kubernetes/minikube).
+* If you are using GKE, please make sure you are using static client certificates before fetching cluster credentials:
+```bash
+gcloud config set container/use_client_certificate True
+gcloud container clusters get-credentials cluster-2 --zone us-central1-a --project istio-demo
+```
+* Ensure curl is present, or clone the github [istio](https://github.com/istio/istio) repository to access the files locally.
 {% endcapture %}
 
 {% capture steps %}
-## Doing ...
+## Install Istio on an existing Kubernetes cluster
+For alpha release, Istio must be installed in the same Kubernetes namespace as the applications. Instructions below will deploy Istio in the "default" namespace. They can be modified for deployment in a separate namespace.
 
-1. Do this.
-1. Do this next. Possibly read this [related explanation](...).
+1. Source Istio configuration file:
+```bash
+source <(curl https://raw.githubusercontent.com/istio/istio/change_install/istio.VERSION)
+```
+2. Install the Istio core components (Istio-Manager, Mixer, Ingress-Controller, and Istio-CA):
+```bash
+kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/kubernetes/istio.yaml
+```
+3. Download one of the [istioctl](../reference/istioctl.md) client binaries corresponding to your OS: `istioctl-osx`, `istioctl-win.exe`,
+`istioctl-linux`, targeted at Mac, Windows or Linux users respectively. For example, run the following commands on a Mac system:
+```bash
+curl ${ISTIOCTL_URL}/istioctl-osx > /usr/local/bin/istioctl
+chmod +x /usr/local/bin/istioctl
+```
+Istioctl is needed to inject Envoy as a side car proxy. It also provides a convenient CLI for creating routing rules and policies.
```

Comment:
"Istioctl CLI is a convenience wrapper around Istio's APIs and provides a simple utility to modify your application's YAML files to add the Envoy sidecar to every pod.".